src/protagonist/DLCS.Model/Assets/IBatchRepository.cs

```diff
@@ -17,8 +17,8 @@ public interface IBatchRepository
     /// <param name="customerId">Id of customer to create batch for.</param>
     /// <param name="assets"></param>
     /// <param name="cancellationToken">Current cancellation token</param>
-    /// <param name="postCreate">Optional action, will be called after Batch is created but before saving in db</param>
+    /// <param name="preCreate">Optional action, will be called after Batch is created but before saving in db</param>

```

Comment:
Should this maintain `postCreate` name? `"will be called after Batch is created but before saving in db"`